### PR TITLE
Fix WiFi Control Panel Issues - Dev-Brainflow

### PR DESCRIPTION
**Pull request by retiutut**
- General Control Panel cleanup for Brainflow integration, specifically WiFi
- Address "Cannot change channel count, sample rate, latency, or protocol for CYTON wifi"

### CODE REVIEW

File: OpenBCI_GUI/ControlPanel.pde [hunk 1]
```diff
@@ -255,6 +228,20 @@ public void controlEvent(ControlEvent theEvent) {
         }
     }
 
+    if (theEvent.isFrom("maxFileDurationCyton")) {
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde]: Putting these here removes a Global function that was in ControlPanel.pde

File: OpenBCI_GUI/ControlPanel.pde [hunk 2]
```diff
@@ -310,12 +297,13 @@ class ControlPanel {
     InterfaceBoxGanglion interfaceBoxGanglion;
     SampleRateCytonBox sampleRateCytonBox;
     SampleRateGanglionBox sampleRateGanglionBox;
-    LatencyCytonBox latencyCytonBox;
-    LatencyGanglionBox latencyGanglionBox;
-    WifiTransferProtcolCytonBox wifiTransferProtcolCytonBox;
-    WifiTransferProtcolGanglionBox wifiTransferProtcolGanglionBox;
     SDBox sdBox;
 
+    //Track Dynamic and Static WiFi mode in Control Panel
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde]: Track this in Control Panel to remove Global variables

File: OpenBCI_GUI/ControlPanel.pde [hunk 3]
```diff
@@ -1108,6 +935,54 @@ class ControlPanel {
                     synthChanButton8.setColorNotPressed(colorNotPressed); //default color of button
                 }
             }
+            
+            //active buttons during Cyton+WiFi/Ganglion+WiFi
```
Comment (retiutut) [inline OpenBCI_GUI/ControlPanel.pde]: This has been moved to check last. Using this with `else if` was breaking buttons in the Control Panel.

**Fixes chief complaint in clubhouse**

File: OpenBCI_GUI/InterfaceHub.pde [hunk 4]
```diff
@@ -244,7 +241,7 @@ class Hub {
     public int tcpBufferPositon = 0;
     private String curProtocol = PROTOCOL_WIFI;
     private String curInternetProtocol = TCP;
-    private String curWiFiStyle = WIFI_DYNAMIC;
+    private String curWiFiStyle = "dynamic"; //this has been moved to ControlPanel class
```
Comment (daniellasry) [inline OpenBCI_GUI/InterfaceHub.pde]: Is this used anywhere or can it be deleted?

Comment (retiutut) [reply OpenBCI_GUI/InterfaceHub.pde]: Yes, I would like to do a PR very soon to remove InterfaceHub.pde entirely!

**Pull request merged**